fre/pp/schema.json

```diff
@@ -3,86 +3,61 @@
   "title": "Schema for PP Yaml",
   "type": "object",
   "properties": {
-    "rose-suite": {
-      "type": "object",
-      "properties": {
-        "settings": {
-          "type":"object",
-          "properties": {
-            "history_segment": {"type":"string"},
-            "pp_start": {"type":"string"},
-            "pp_stop": {"type":"string"},
-            "site": {"type":"string"},
-            "fre_analysis_home": {"type":["string","null"]},
-            "pp_chunk_a": {"type":"string"},
-            "pp_chunk_b": {"type":["string","null"]},
-            "pp_grid_spec": {"type":"string"}
-          }
-        },
-        "switches": {
-          "type":"object",
-          "properties": {
-            "clean_work": {"type":"boolean"},
-            "do_mdtf": {"type":"boolean"},
-            "do_statics": {"type":"boolean"},
-            "do_timeavgs": {"type":"boolean"},
-            "do_refinediag": {"type":"boolean"},
-            "do_atmos_plevel_masking": {"type":"boolean"},
-            "do_preanalysis": {"type":"boolean"},
-            "do_analysis": {"type":"boolean"},
-            "do_analysis_only": {"type":"boolean"}
-          }
-        },
-        "directories": {
-          "type":"object",
-          "properties": {
-            "history_dir": {"type":"string"},
-            "pp_dir": {"type":"string"},
-            "ptmp_dir": {"type":"string"},
-            "refinediag_scripts":{"type":["string","null"]},
-            "preanalysis_script":{"type":["string","null"]},
-            "history_refined":{"type":["string","null"]},
-            "analysis":{"type":["string","null"]}
-          }
-        },
-        "metadata": {
-          "type":"object",
-          "properties": {
-            "experiment":{"type":"string"},
-            "platform":{"type":"string"},
-            "target":{"type":"string"}
-          }
-        }
-      }
-    },
-    "component": {
-      "type":"object",
-      "properties": {
-        "type": {"type":"string"},
-        "sources": {"type":"string"},
-        "sourceGrid": {"type":"string"},
-        "xyInterp": {"type":"string"},
-        "interpMethod": {"type":"string"},
-        "inputRealm": {"type":"string"}
-      }
-    },
-    "tmpdir": {
-      "type": "object",
-      "properties": {
-        "tmpdirpath": {"type":["string","null"]}
-      }
-    },
-    "install-exp-script": {
-      "type": "array",
-      "properties": {
-        "path": {"type": "string"},
-        "install-option": {
-          "type": "object",
-          "properties": {
-            "install": {"type": "string"}
-          }
-        }
-      }
-    }
+       "name": {"type": "string"},
```

Comment:
We want this schema to be a submodule of https://github.com/NOAA-GFDL/gfdl_msd_schemas soon, but since this works, I think we should take this as is and refactor in another PR.

Reply:
I agree